write-stats.js

```diff
@@ -15,7 +15,7 @@ function writeStats(emitter, options) {
             clientName,
             resource,
             'request'
-        ]);
+        ].join('.'));
```

Comment:
Let’s take note that in the future, we would like to move away from the `[part, part].join('.')` pattern to `part + '.' + part` since the latter is fast in V8.